src/balatro_gym/cards/joker/joker.py

```diff
@@ -0,0 +1,303 @@
+
+
+class JollyJoker(JokerBase):
+    _cost: int = 3
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_mult(self, scored_cards: Sequence[PlayingCard], state: BlindState, scored_hand: PokerHandType) -> int:
+        if contains_one_pair(get_max_rank(state.hand)):
+            return 8
+        return 0
+
+
+class ZanyJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_mult(self, scored_cards: Sequence[PlayingCard], state: BlindState, scored_hand: PokerHandType) -> int:
+        if contains_three_set(state.hand):
+            return 12
+        return 0
+
+
+class MadJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_mult(self, scored_cards: Sequence[PlayingCard], state: BlindState, scored_hand: PokerHandType) -> int:
+        if contains_two_pair(get_max_rank(state.hand)):
+            return 10
+        return 0
+
+
+class CrazyJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_mult(self, scored_cards: Sequence[PlayingCard], blind: BlindState, scored_hand: PokerHandType) -> int:
+        if (
+            scored_hand == PokerHandType.STRAIGHT
+            or scored_hand == PokerHandType.STRAIGHT_FLUSH
+            or scored_hand == PokerHandType.ROYAL_FLUSH
+        ):
+            return 12
+        return 0
+
+
+class DrollJoker(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_mult(self, scored_cards: Sequence[PlayingCard], blind: BlindState, scored_hand: PokerHandType) -> int:
+        if (
+            scored_hand == PokerHandType.FLUSH
+            or scored_hand == PokerHandType.FLUSH_FIVE
+            or scored_hand == PokerHandType.FLUSH_HOUSE
+            or scored_hand == PokerHandType.STRAIGHT_FLUSH
+        ):
+            return 10
+        return 0
+
+
+class SlyJoker(JokerBase):
+    _cost: int = 3
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.CHIPS
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_chips_hand(
+        self, scored_cards: Sequence[PlayingCard], blind: BlindState, board: BoardState, scored_hand: PokerHandType
+    ) -> int:
+        return 50 if get_num_pairs(scored_cards) >= 1 else 0
```

Comment:
nit: The JollyJoker has the same condition "played hand contains pair" but we're using a different function (`contains_one_pair(get_max_rank(state.hand))`). We should probably unify